packages/truffle-reporters/compile/index.js

```diff
@@ -0,0 +1,46 @@
+const OS = require("os");
+
+module.exports = {
+  handlers: {
+    "compile:startJob": function() {
```

Comment:
If we go with standardizing on verbs like `start` and `finish`, then "job" becomes redundant... the entirety of "compile" is between `compile:start` and `compile:finish` (or `compile:error`)... profiling happens during that process and would be between `compile:profile:start` and `compile:profile:finish`.

If we furthermore stick with only imperative mood, it's shorter to type and consistent: `compile:profile:warn`, etc., and then easier to understand at a glance when looking at a streamed list of event names.